src/plugins/discover/public/application/main/utils/fetch_all.ts

```diff
@@ -90,6 +90,7 @@ export function fetchAll(
     () => sendCompleteMsg(dataSubjects.main$, true),
```

Comment:
Here's how it should work:

in L85 you need to add a pipe to catch the error if fetchChart fails, like:

fetchChart(dataSubjects, searchSource.createCopy(), subFetchDeps).pipe(catchError(err => null))

then this error would not be propagated and the chart error message would be displayed